dev-vcs/monotone/monotone-1.0-r5.ebuild

```diff
@@ -2,23 +2,24 @@
 # Distributed under the terms of the GNU General Public License v2
 
 # QA failiures reported in https://code.monotone.ca/p/monotone/issues/181/
-EAPI="4"
-inherit bash-completion-r1 elisp-common eutils toolchain-funcs user
+EAPI=6
+inherit bash-completion-r1 elisp-common tmpfiles user
 
 DESCRIPTION="Monotone Distributed Version Control System"
 HOMEPAGE="http://monotone.ca"
-SRC_URI="http://monotone.ca/downloads/${PV}/${P}.tar.bz2"
+SRC_URI="http://monotone.ca/downloads/${PV}/${P}.tar.bz2
+	http://dev.gentoo.org/~axs/distfiles/${P}-patches.tar.xz"
 
 LICENSE="GPL-2"
 SLOT="1"
 KEYWORDS="amd64 ~ia64 x86"
```

Comment:
need all ~, if you bump EAPI

Reply:
Why??  Files-on-disk isn't changing (except for the /var/run permanency fix), just VDB, and this is a stable EAPI so nothing to worry about there either...